GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -122,14 +122,14 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                         // service
                         this.CopyAllFiles(
-                            GVFSPlatform.Instance.GetDataRootForGVFS(),
+                            GVFSPlatform.Instance.GetCommonAppDataRootForGVFS(),
                             archiveFolderPath,
                             this.ServiceName,
                             copySubFolders: true);
 
                         // service ui
                         this.CopyAllFiles(
-                            GVFSPlatform.Instance.GetDataRootForGVFS(),
+                            GVFSPlatform.Instance.GetCommonAppDataRootForGVFS(),

```

Comment:
Do we need additional calls to copy the service and service UI logs? (they are staying in their old location)

Reply:
No need for additional calls - there is no change in how the Logs appear in diagnostics zip file. Here is the structure (it is same as the old).

<img width="173" alt="Screen Shot 2019-12-06 at 7 21 48 AM" src="https://user-images.githubusercontent.com/378580/70322883-53e47c80-17f9-11ea-9ed2-58533fa7bcc5.png">

Reply:
Ahh, I had these mixed up.  Previously we'd also capture the config file in `gvfs diagnose` and now we will not.  Can/should we still copy that over when `diagnose` is run?

I also have trouble keeping the difference between `GetDataRootForGVFS` and `GetCommonAppDataRootForGVFS` straight.  Are there any other method names we could use that would be more descriptive?

Reply:
>>Previously we'd also capture the config file in gvfs diagnose and now we will not. Can/should we still copy that over when diagnose is run?

We still capture `gvfs.config` file - the previous screenshot is showing only the directories (not files). Here is the one with root level files as well - 
<img width="403" alt="Screen Shot 2019-12-06 at 12 42 43 PM" src="https://user-images.githubusercontent.com/378580/70343743-11856480-1826-11ea-8892-7b396eec4c2d.png">

>>I also have trouble keeping the difference between GetDataRootForGVFS and GetCommonAppDataRootForGVFS straight. Are there any other method names we could use that would be more descriptive?

How about `GetSecureAppDataRootForGVFS` and `GetCommonAppDataRootForGVFS`

Reply:
> We still capture `gvfs.config` file - the previous screenshot is showing only the directories (not files). Here is the one with root level files as well

Where is this getting copied by `diagnose`?  Doesn't the config file live in `GetSecureAppDataRootForGVFS()`?

I'm wondering if you have an old gvfs.config file on your machine that was not migrated into ProgramFiles.

Reply:
Take a look at [`DiagnoseVerb.cs` line no. 163.](https://github.com/microsoft/VFSForGit/pull/1597/commits/e3b2be03cc53222bff9c51cf8fb9ea0470f03566#diff-fd719e3f262436196484b5920d660d5d)